WatchParty/client/src/components/SearchResults/SearchResults.js

```diff
@@ -75,12 +77,14 @@ class SearchResults extends Component{
                         <img onError={this.defaultPoster} src={`https://image.tmdb.org/t/p/w1280${el.poster_path}`} width="110px" height="190px" alt="movie poster"/>
                       </div>
                       <div className="title-desc-container">
-                        <div className="movie-title">
-                          {el.release_date !== undefined && el.release_date !== "" ? (<div> {el.title} ({el.release_date.substring(0,4)})</div> ) : (<div> {el.title} </div> )}
-                        </div>
-                          <div className="movie-desc">
-                            <div> {el.overview} </div>
+                        <Link to={`/results/${el.id}`}>
+                          <div className="movie-title">
+                            {el.release_date !== undefined && el.release_date !== "" ? (<div> {el.title} ({el.release_date.substring(0,4)})</div> ) : (<div> {el.title} </div> )}
```

Comment:
This is more of a linting thing; soon we'll have a linter and this might break the "line length" rule